DependencyInjection/LiipMonitorExtension.php

```diff
@@ -255,6 +254,26 @@ private function configureDoctrineMigrationsCheck(ContainerBuilder $container, a
         }
     }
 
+    private function configureMailer(ContainerBuilder $container, LoaderInterface $loader, array $config)
+    {
+        if (false === $config['mailer']['enabled']) {
+            return;
+        }
+
+        try {
+            $mailerDefinition = $container->findDefinition('mailer');
+        } catch (ServiceNotFoundException $e) {
+            throw new MissingPackageException('To enable mail reporting you have to install the "swiftmailer/swiftmailer" or "symfony/mailer".');
```

Comment:
Can we just throw a \RuntimeException here? Don't know that we need a custom exception for this.